apps/map/Dockerfile

```
@@ -1,84 +1,67 @@
 FROM --platform=linux/amd64 node:24.14.1-alpine AS builder
 ARG TURBO_VERSION
 
-ENV NEXT_TELEMETRY_DISABLED 1
-ENV TURBO_TELEMETRY_DISABLED 1
-
-RUN apk add --no-cache libc6-compat openssl && apk update
+ENV TURBO_TELEMETRY_DISABLED=1
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+ENV COREPACK_ENABLE_DOWNLOAD_PROMPT=0
+RUN apk add --no-cache libc6-compat && apk update
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)